packages/@react-aria/listbox/docs/useListBox.mdx

```diff
@@ -333,18 +416,16 @@ For options that have more complex content (e.g. icons, multiple lines of text,
 as needed to apply to the main text element of the option and its description. This improves screen
 reader announcement.
 
-**NOTE: listbox options cannot contain interactive content (e.g. buttons, checkboxes, etc.).
-For these cases, see `useGrid` instead.**
+**NOTE: listbox options cannot contain interactive content (e.g. buttons, checkboxes, etc.).**
 
-This example shows how `labelProps` and `descriptionProps` can be applied to child elements of
-the item to apply ARIA properties returned by <TypeLink links={docs.links} type={docs.exports.useOption} />.
-This is done using `React.cloneElement` in this example, but you can use context or other
-approaches for this as well.
+To implement this, we'll update the `Option` component to apply the ARIA properties
```

Comment:
This example should probably be single selection only based on the options in the box